libs/sdk-ui-dashboard/src/model/commandHandlers/drill/drillToAttributeUrlHandler.ts

```diff
@@ -30,11 +31,18 @@ export function* drillToAttributeUrlHandler(
         ctx,
     );
 
+    const filtersInfo: SagaReturnType<typeof getDrillToUrlFiltersWithResolvedValues> = yield call(
```

Comment:
Detail: can we run the new `filtersInfo` call in parallel with the `resolvedUrl` call? It could speed things up.

Reply:
Same in the drill to custom URL handler.

Reply:
Good point, will update